src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -29,7 +31,8 @@ export const buildCpsAssetPageATIParams = (
   return {
     appName: atiAnalyticsAppName,
     contentId: path(['id'], metadata),
-    contentType: 'article-media-asset',
+    contentType:
+      type === 'PGL' ? 'article-photo-gallery' : 'article-media-asset',
```

Comment:
What do you think about passing the `contentType` into `buildParams`, from `CpsPgl`, `CpsSty` and `CpsMap`? This allows you to follow the [Open-Closed principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle)

Being more specific, it means this component can be left unchanged, but each page type can specify a content type that suits it. Also, following the principle any number of page types could be added without modifying this component.

Reply:
Done!